src/table/components/PaginationContent.jsx

```diff
@@ -160,11 +159,13 @@ export default function PaginationContent({
       {showFirstAndLast && getButton(onFirstPage, 0, 'FirstPage')}
       {getButton(onFirstPage, page - 1, 'PreviousPage')}
       {getButton(onLastPage, page + 1, 'NextPage', !showFirstAndLast ? handleLastButtonTab : null)}
-      {showFirstAndLast && getButton(onLastPage, lastPageIdx, 'LastPage', handleLastButtonTab)}
+      {showFirstAndLast && getButton(onLastPage, totalPages - 1, 'LastPage', handleLastButtonTab)}
     </>
   );
 }
 
+export default memo(PaginationContent);
```

Comment:
in other files, you put this line below defaultProps, right?

Reply:
you are right, I'll move it